lib/src/rules/public_member_api_docs.dart

```diff
@@ -283,6 +285,57 @@ class _Visitor extends SimpleAstVisitor {
     }
   }
 
+  @override
+  visitExtensionDeclaration(ExtensionDeclaration node) {
```

Comment:
This should also check (static) fields.

Reply:
They get checked in `visitFieldDeclaration`.  (I've added a testcase too.)